### PR TITLE
[C-3038] Remove playlist updates feature flags

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Removes playlist-improvements feature flags and dead code. This will make it much easier to fix an issue related to uploaded cover-art getting rotated on mobile, since i wont have to update legacy code

### Testing

Went through all the new playlist flows to confirm things still work as expected

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-3038-playlist-updates-ff

File: packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx [hunk 1]
```diff
@@ -89,7 +70,7 @@ export const PlaylistLibrary = (props: PlaylistLibraryProps) => {
         <CreatePlaylistLibraryItemButton scrollbarRef={scrollbarRef} />
       </GroupHeader>
       {!library || isEmpty(library?.contents) ? (
-        <EmptyLibraryNavLink onClick={handleCreatePlaylist} />
+        <EmptyLibraryNavLink />
```
Comment (Kyle-Shanks) [inline packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx:73]: cant remember which component this is, but do we want to do something else on click, or remove click interaction all together?

**Review by Kyle-Shanks — APPROVED** [review]
BIG. great to see all the red

File: packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx:73]: the click is handled in the component itself now!

**Pull request merged**